src/content/configuration/other-options.md

```diff
@@ -126,6 +126,26 @@ module.exports = {
 
 W> The final location of the cache is a combination of `cache.cacheDirectory` + `cache.name`.
 
+### `cache.cacheLocation`
+
+`string`
+
+Locations for the cache. Defaults to `cache.cacheDirectory` / `cache.name`.
```

Comment:
```suggestion
Locations for the cache. Defaults to `cache.cacheDirectory` or `cache.name`.
```

Reply:
IG it is `cache.cacheDirectory/cache.name`

Reply:
Maybe it is better to writing it like JS template literals?`${cache.cacheDirectory}/${cache.name}`

Not sure.

Reply:
+1 for that :)

Reply:
Yeah Fernando's suggestion looks good although we could even show the default including path call `path.resolve(cache.cacheDirectory, cache.name)` to be on the safe side with odd paths given WDYT?